tests/270_imagick_restoreErrorHandler.phpt

```diff
@@ -16,7 +16,7 @@ catch(ImagickException $ie) {
 }
 
 try {
-	$x = @(5 / 0);
+	$x = @$x;
```

Comment:
but here's no longer division here... i'm missing context?

Reply:
Yes, because this test covers warning